src/ImageRunModal.jsx

```diff
@@ -463,13 +471,20 @@ export class ImageRunModal extends React.Component {
     }
 
     clearImageSelection = () => {
+        // Reset command if it was prefilled
+        let command = this.state.command;
+        if (this.state.selectedImage && this.state.selectedImage.Command && this.state.command === utils.quote_cmdline(this.state.selectedImage.Command))
+            command = "";
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1155-20230104-111228-5fe35f89-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Can you please add a test for this?

Also, this is an excellent case for using [coalescing operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), i. e. 

    if (this.state.selectedImage?.Command ..)